addons/barcodes_gs1_nomenclature/i18n/barcodes_gs1_nomenclature.pot

```diff
@@ -160,16 +160,6 @@ msgstr ""
 msgid "The FNC1 Seperator Alternative is not a valid Regex: "
```

Comment:
"Seperator" should be "Separator", but I was too lazy to open PR just to fix the typo where it shows up. May as well fix it as part of this PR?